src/main/java/org/openrewrite/staticanalysis/InstanceOfPatternMatch.java

```diff
@@ -224,6 +224,21 @@ public J.InstanceOf processInstanceOf(J.InstanceOf instanceOf, Cursor cursor) {
                     name,
                     type,
                     null));
+            JavaType.FullyQualified fqType = TypeUtils.asFullyQualified(type);
+            if (fqType != null && !fqType.getTypeParameters().isEmpty() && !(instanceOf.getClazz() instanceof J.ParameterizedType)) {
+                TypedTree oldTypeTree = (TypedTree) instanceOf.getClazz();
+
+                J.ParameterizedType newTypeTree = new J.ParameterizedType(
+                        randomId(),
+                        oldTypeTree.getPrefix(),
+                        Markers.EMPTY,
+                        oldTypeTree.withPrefix(Space.EMPTY),
+                        null,
+                        oldTypeTree.getType()
+                ).withTypeParameters(fqType.getTypeParameters().stream().map(p -> new J.Wildcard(randomId(), Space.EMPTY, Markers.EMPTY, null, null)).collect(Collectors.toList())
```

Comment:
Note to self : look at listutils to conform with code elsewhere.

Reply:
`ListUtil.map()` is for the same type lists, i.e. mapping of `List<J>` -> `List<J>` in my case it is `List<JavaType>` -> `List<J>`. I felt it is okay... the list can't be long... I only care about its size. If there are 5 type parameters i wanted to produce `<?,?,?,?,?>. I can rerwrite this piece to conform to what i mentioned above better rather than have `map`